modules/contact/src/main/kotlin/edu/stanford/spezi/modules/contact/ContactView.kt

```diff
@@ -14,55 +15,52 @@ import androidx.compose.foundation.layout.size
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.AccountBox
 import androidx.compose.material3.Card
+import androidx.compose.material3.HorizontalDivider
 import androidx.compose.material3.Icon
 import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.collectAsState
-import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.tooling.preview.PreviewParameter
-import androidx.compose.ui.tooling.preview.PreviewParameterProvider
 import edu.stanford.spezi.core.design.theme.Sizes
 import edu.stanford.spezi.core.design.theme.Spacings
 import edu.stanford.spezi.core.design.theme.SpeziTheme
 import edu.stanford.spezi.core.design.theme.TextStyles
+import edu.stanford.spezi.modules.contact.component.AddressCard
 import edu.stanford.spezi.modules.contact.component.ContactOptionCard
-import edu.stanford.spezi.modules.contact.component.NavigationCard
 import edu.stanford.spezi.modules.contact.model.Contact
-import edu.stanford.spezi.modules.contact.repository.DefaultContactRepository
+import edu.stanford.spezi.modules.contact.model.ContactOption
+import edu.stanford.spezi.modules.contact.model.PersonNameComponents
+import edu.stanford.spezi.modules.contact.model.call
+import edu.stanford.spezi.modules.contact.model.email
+import edu.stanford.spezi.modules.contact.model.website
+import java.util.Locale
 
 /**
  * ContactView composable function to display contact information.
  *
- * @param viewModel The ViewModel associated with the screen.
+ * @param contact The contact associated with the view.
  *
- * @sample edu.stanford.spezi.modules.contact.ContactScreenPreview
+ * @sample edu.stanford.spezi.modules.contact.ContactViewPreview
  *
  * @see Contact
- * @see ContactOptionCard
- * @see NavigationCard
- * @see edu.stanford.spezi.modules.contact.repository.ContactRepository
+ * @see ContactOptionButton
+ * @see AddressButton
  */
 @OptIn(ExperimentalLayoutApi::class)
 @Composable
-fun ContactScreen(viewModel: ContactViewModel) {
-    val contact by viewModel.contact.collectAsState()
-
+fun ContactView(contact: Contact) {
```

Comment:
I changed the order of modifier and contact to make it possible to create the ContactComposable with only specifying the contact and without parameter names, otherwise I like those changes!